specs/airr-schema.yaml

```diff
@@ -2259,118 +2253,152 @@ Clone:
         clone_id:
             type: string
             description: Identifier for the clone.
+            x-airr:
+                miairr: false
+                required: true
         data_processing_id:
             type: string
             description: Identifier for the data processing object.
+            x-airr:
+                miairr: false
         v_call:
             type: string
             description: V gene with allele. For example, IGHV4-59*01.
             example: IGHV4-59*01
+            x-airr:
+                miairr: false
+                required: true
         d_call:
             type: string
             description: D gene with allele. For example, IGHD3-10*01.
             example: IGHD3-10*01
+            x-airr:
+                miairr: false
+                required: true
         j_call:
             type: string
             description: J gene with allele. For example, IGHJ4*02.
             example: IGHJ4*02
+            x-airr:
+                miairr: false
+                required: true
         junction:
             type: string
             description: >
                 Junction region nucleotide sequence, where the junction is defined as
                 the CDR3 plus the two flanking conserved codons.
+            x-airr:
+                miairr: false
         junction_aa:
             type: string
             description: >
                 Junction region amino acid sequence.
+            x-airr:
+                miairr: false
         junction_length:
             type: integer
             description: Number of nucleotides in the junction sequence.
+            x-airr:
+                miairr: false
         junction_aa_length:
             type: integer
             description: Number of amino acids in the junction sequence.
+            x-airr:
+                miairr: false
         germline_alignment:
             type: string
             description: >
                 Assembled, aligned, full-length inferred germline sequence spanning the same region
                 as the sequence_alignment field (typically the V(D)J region) and including the same set
                 of corrections and spacers (if any).
```

Comment:
It seems like this definition has been copied from the rearrangements schema, but I don't think it makes sense in this context. To avoid confusion, I would recommend changing the field name, as well. Maybe:
```
naive_ancestor:
    type: string
    description: Full-length inferred sequence of the naive ancestor of this clone.
```
Also, I think it should be a required field?

Related, `germline_alignment_aa` below should become `naive_ancestor_aa`

Reply:
Looks like this never got addressed. It makes sense to me, so hopefully @scharch is able to commit directly to this branch now? I think @javh was going to try to fix whatever was preventing that but not sure if that happened yet. If not I can try

Reply:
I think it got settled -we left the names as they were but I updated the descriptions in #302

Reply:
Yeah, @scharch and I resolved this.